src/app/pages/tech/tech.component.html

```diff
@@ -1,5 +1,8 @@
 <app-main-nav pageTitle={{title}}></app-main-nav>
 <div class="container">
+    <input [(ngModel)]="className.isSelected" *ngFor="let className of class " type="checkbox" name="{{className}}"
```

Comment:
can we remove this series of inputs for the commit?
i think we will definitely use this feature for other columns on other data, but i think that there are too many jobs (classes) to use it here.